blueprints/ember-frost-list/index.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-theme', target: '^1.3.0'}
+        {name: 'ember-frost-core', target: '0.0.13 - 1'}
```

Comment:
This is basically saying the version must be greater than or equal to 0.0.13 but less than 2.0.0.